### DISCUSSION TITLE
Configuring static web app using azure portal

### DISCUSSION THREAD

**Discussion by rexkenley** [Azure/static-web-apps#1453] [Q&A] [answered]
Hi

I am trying to set the globalHeaders listed in this blog using azure portal
https://blog.jonsdocs.org.uk/2023/09/05/azure-static-web-apps-and-http-security-headers/

![image](https://github.com/Azure/static-web-apps/assets/10548251/adbfef18-5d91-440b-93d4-0fa819190f02)
![image](https://github.com/Azure/static-web-apps/assets/10548251/c2ec2643-eb08-4905-add4-66bfc12ba249)

How do I do that? Thanks

**Comment by johnnyreilly** [comment]
You don't. You use a `staticwebapp.config.json` file

**Reply by rexkenley** [reply]
@johnnyreilly 

Do you have an example on how to add it?

I am publishing a  [storybook static site](https://storybook.js.org/docs/sharing/publish-storybook#build-storybook-as-a-static-web-application), but have NO idea on how to add a staticwebapp.config.json after uploading the artifacts to azure.

**Reply by johnnyreilly** [reply]
It should be part of your repository. Place it in the root of your SWA and that should probably work - like this: https://github.com/johnnyreilly/blog.johnnyreilly.com/blob/main/blog-website/staticwebapp.config.json

**Comment by rexkenley** [comment]
@johnnyreilly 

Is there a way to use a pipeline action to manually add that file? I cannot add files to the storybook build result so I have to add it afterwards.

**Reply by johnnyreilly** [reply, marked as answer]
You can just use bash - see here: https://github.com/johnnyreilly/blog.johnnyreilly.com/blob/22520d120c68fadf8cb88103b6ab4f85fa3f3660/.github/workflows/build-and-deploy-static-web-app.yml#L168

**Comment by rexkenley** [comment]
@johnnyreilly 
![image](https://github.com/Azure/static-web-apps/assets/10548251/201ad60c-268d-4a60-8d1c-adbe9bcd1eab)
